packages/instant/src/components/erc20_token_selector.tsx

```diff
@@ -35,7 +35,7 @@ export class ERC20TokenSelector extends React.Component<ERC20TokenSelectorProps>
                     value={this.state.searchQuery}
                     onChange={this._handleSearchInputChange}
                 />
-                <Container overflow="scroll" height="275px" marginTop="10px">
+                <Container overflow="scroll" height={{ default: '275px', sm: '75vh' }} marginTop="10px">
```

Comment:
`75vh` feels like an arbitrary number here, I essentially want this to be "100%" but if I set it to that, then the  `overflow: scroll;` doesn't work.

Reply:
Yea... still not sure whether it's possible to make scrolling work without a fixed height here?  

Using`vh` here makes sense to me. If we know the exact offset from the bottom maybe something like `calc(100vh - 100px)` makes sense too.